docs/website/guides/explain-systems.md

```diff
@@ -101,8 +101,8 @@ query @explain {
                     "collectionID": "3",
                     "collectionName": "Author",
                     "spans": [{
-                        "start": "/3",
-                        "end": "/4"
+                        "start": "/\x8b",
```

Comment:
nitpick: My eyes 😮

Reply:
Has been reverted - encoding is not json friendly, so explain must be pretty-printed anyway